maxima/g0/deflate_geo/deflate-geo.mac

```diff
@@ -0,0 +1,72 @@
+load("modal-basis")$
+load(stringproc)$
+load("out-scripts")$
+
+/* Taking DG fields g_ij and |B| as inputs, this kernel will calculate the other geometric quantities
+J = det(g_ij))
+b_i = g_iz/sqrt(g_zz)
+*/
+
+deflateGeo(fh, basisName, polyOrder) := block(
```

Comment:
@akashukla please add the list of all variables/lists/objects private to this function following `block(`, see for example
https://github.com/ammarhakim/gkylcas/blob/507993efded0ea3c452e725a962bd44f2b2b720b/maxima/g0/fem-poisson/fem-poisson-sol_stencil.mac#L12

one way to do this is to read your function and just type out all the variables. Another is to run the function and then run `values`, which prints the name of the variables leaked. But it'll print ALL the variables leaked, not just the ones from your code, so you have to know which ones to pick out

Reply:
this applies to all the functions in this PR